contrib/depends/Makefile

```
@@ -3,7 +3,7 @@
 SOURCES_PATH ?= $(BASEDIR)/sources
 BASE_CACHE ?= $(BASEDIR)/built
 SDK_PATH ?= $(BASEDIR)/SDKs
-FALLBACK_DOWNLOAD_PATH ?= https://bitcoincore.org/depends-sources
+FALLBACK_DOWNLOAD_PATH ?= https://downloads.sourceforge.net/project/boost/boost/1.73.0/
```

did you check link for correctness ?
```suggestion
FALLBACK_DOWNLOAD_PATH ?= https://sourceforge.net/projects/boost/files/boost/1.73.0/
```